build.gradle.kts

```diff
@@ -1,30 +1,7 @@
 plugins {
-	id("org.springframework.boot") version "3.2.4" apply false
-	id("io.spring.dependency-management") version "1.1.4" apply false
```

Comment:
we don't need it anymore, since we use it in custom one. In addition, gradle does not allow to set versions in 2 places, so we must remove it